### PR TITLE
New version: PeterPawlowski.foobar2000.EncoderPack version 2024-03-03

### LINKED ISSUE #142186: [Update Request]: PeterPawlowski.foobar2000.EncoderPack
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

PeterPawlowski.foobar2000.EncoderPack

### Package Version

2024-03-03

### Please describe the changes you would like to see

There is a new version available: https://www.foobar2000.org/encoderpack

https://www.foobar2000.org/getfile/Free_Encoder_Pack-2024-03-03.exe

### PR DESCRIPTION
**Pull request by Exorcism0666**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

* Resolves #142186

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/142193)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   DEBDAF4C9BFC6DB3B364278F1366A500D730C505531FEB6D96AC1BE1AD9BDA92
 Expected hash: 40c53eb1d82d7eb6eef4e594882d94ac229c88597629723507f2f8c44556c362
 
 3/5/2024 12:46:33 PM
 
 ****************************************************************************************
 Installer Validation
 ****************************************************************************************
 

(Automated response - build 861.)

File: manifests/p/PeterPawlowski/foobar2000/EncoderPack/2024-03-03/PeterPawlowski.foobar2000.EncoderPack.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created with YamlCreate.ps1 v2.3.4 $debug=QUSU.CRLF.5-1-19041-4123.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: PeterPawlowski.foobar2000.EncoderPack
+PackageVersion: 2024-03-03
+InstallerType: nullsoft
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.foobar2000.org/getfile/Free_Encoder_Pack-2024-03-03.exe
+  InstallerSha256: 40C53EB1D82D7EB6EEF4E594882D94AC229C88597629723507F2F8C44556C362
```
Comment (stephengillie) [inline manifests/p/PeterPawlowski/foobar2000/EncoderPack/2024-03-03/PeterPawlowski.foobar2000.EncoderPack.installer.yaml:10]:
```suggestion

```

(Automated response - build 861.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-03-06T00:35:35.6291365Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Installers][0] Description: Missing required property 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #0 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: PeterPawlowski.foobar2000.EncoderPack.installer.yaml Manifest Error: Required field missing. [InstallerSha256]

(Automated response - build 861.)

**Review by stephengillie — APPROVED** [review]

**Comment by Exorcism0666** [comment]
@Trenly

**Comment by Trenly** [comment]
* https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**